base/sort.jl

```diff
@@ -432,6 +443,30 @@ providing a custom "less than" function; use `rev=true` to reverse the sorting o
 options are independent and can be used together in all possible combinations: if both `by`
 and `lt` are specified, the `lt` function is applied to the result of the `by` function;
 `rev=true` reverses whatever ordering specified via the `by` and `lt` keywords.
+
+```jldoctest
+julia> v = [3, 2, 1]; sort!(v); v
+3-element Array{Int64,1}:
+ 1
+ 2
+ 3
+
+julia> v = [3, 2, 1]; sort(v, rev = true); v
```

Comment:
should be `sort!` here, and also probably shouldn't arrange `v` so that `rev = true` version is a noop